recipes/pyam/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "2348431559b86d8e719bcbfe6bf5cc88b0ffbabdc64fc6ed5e34c55d67c58310" %}
+
+package:
+  name: pyam
+  version: {{ version }}
+
+source:
+  url: https://github.com/IAMconsortium/pyam/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
This seems like it could be `noarch: python` or does it do some undeclared compilation?